roles/tests-integration/lib/sniffer.rs

```diff
@@ -587,6 +625,65 @@ impl Sniffer {
             }
         }
     }
+
+    /// Converts a message type number to a human-readable name
+    fn message_type_to_name(msg_type: MsgType) -> &'static str {
```

Comment:
I wonder if this should be part of itf or part of roles-logic or consts crates..